plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java

```diff
@@ -133,6 +142,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns ListTag(NPCTag)
         // @description
         // Gets a list of all NPCs currently within the area.
+        // @Example
+        // # Narrates a list of NPCs' names that are within the area separated by a comma and a space.
+        // - narrate "List of NPCs in 'my_cuboid': <cuboid[my_cuboid].npcs.parse[name].separated_by[, ]"

```

Comment:
This is missing the `>` at the end.

Make sure you actually test your examples.